autorgb/ProcessImages.py

```diff
@@ -100,6 +136,34 @@ def process_file(self, file, destination):
             image_number += 1
 
     def update_progress(self):
+        """
```

Comment:
This docblock is incorrect. It looks like it's the docblock for get_pixel(). This one is supposed to update the progress bar in the tkinter window.